prime-router/src/main/kotlin/azure/SettingsFunctions.kt

```diff
@@ -266,14 +262,9 @@ open class BaseFunction(
         settingName: String,
         clazz: Class<T>,
         organizationName: String? = null,
-        oktaSender: Boolean = false
     ): HttpResponseMessage {
-        return oktaAuthentication.checkAccess(request, organizationName ?: settingName, oktaSender) {
-            // if the user is an okta sender, their organization name matches the okta group naming schema
-            // it would be something like "ignore.ignore-waters", where "ignore" is the organization name
-            // split the organizationName to get the correct organization to find in the settings database
-            val settingValue = if (oktaSender) settingName.split(".")[0] else settingName
-            val setting = facade.findSettingAsJson(settingValue, clazz, organizationName)
+        return oktaAuthentication.checkAccess(request, organizationName ?: settingName) {
+            val setting = facade.findSettingAsJson(settingName, clazz, organizationName)
```

Comment:
Revmoed this split stuff.

There's no use case in which we should ever get our dot-format orgname.senderName as a name here.  The API syntax has these separated, eg, "/organization/ignore/sender/ignore-waters".